### ISSUE TITLE
at_onboarding_flutter refactoring

### ISSUE THREAD

**Issue by cconstab** [atsign-foundation/at_client_sdk#1615]
### Now we know what this widget should do..

It's time to refactor and make the code much more readable.

- [ ] Take stock of what the widget needs to do
- [ ] Include the removal of the QRCode button (using the Registrar API we should be able to pop that up if required)
- [ ] Outline the architecture in the architecture call
- [ ] Implement
- [ ] test
- [ ] deploy
- [ ] repeat

**Comment by gkc** [comment]
@cconstab a few months have passed ... is it time to take a look at this again?

**Comment by sarika01** [comment]
This can be taken up along with UX change. 
https://github.com/atsign-foundation/at_widgets/issues/321

**Comment by ksanty** [comment]
@sachins-geekyants @nitesh2599 not sure the status of this ticket.  Looks like this might be completed based on ticket above?

**Comment by sachins-geekyants** [comment]
@ksanty yes, It have been addressed, we are in process of releasing new at_onboarding_flutter version.
